grails-app/domain/org/pih/warehouse/inventory/TransactionType.groovy

```diff
@@ -39,4 +39,9 @@ class TransactionType implements Serializable {
         String regex = "\\${Constants.LOCALIZED_STRING_SEPARATOR}"
         return name.split(regex)[0] == transactionTypeName.split(regex)[0]
     }
+
+    // Checking whether the transaction type is adjustment for displaying the appropriate icon on the stock history page
```

Comment:
@alannadolny I meant rather to put a comment here "**why** it's determined by doing `compareName(Constants.ADJUSTMENT)`", not "for what it is used"